### PR TITLE
Fix issues in Makefile.PL

### PR DESCRIPTION
**Pull request by pali**

### CODE REVIEW

File: Makefile.PL [hunk 1]
```diff
@@ -464,11 +464,7 @@ my %o = ( 'NAME' => 'DBD::MariaDB',
 	  'OBJECT' => '$(O_FILES)',
 	  'LIBS' => (join ' ', @libdirs, $main_lib, @libs),
 	  @ldflags ? (dynamic_lib => { OTHERLDFLAGS => (join ' ', @ldflags) }) : (),
-	  'VERSION_FROM' => 'lib/DBD/MariaDB.pm'
-	);
-
-if (eval { ExtUtils::MakeMaker->VERSION(5.43) }) {
-  %o=(%o,
+	  'VERSION_FROM' => 'lib/DBD/MariaDB.pm',
```
Comment (choroba) [inline Makefile.PL:468]: This breaks indentation if tab width != 2.

Comment (pali) [reply Makefile.PL:468]: All lines before `'VERSION_FROM'` have same style of indentation. (Ehm... it is a style? :D)

The only change which I done for this `'VERSION_FROM'` line was adding a trailing comma. No other whitespace change.

Comment (choroba) [reply Makefile.PL:468]: Yes, but the following lines have a different "style".

Comment (pali) [reply Makefile.PL:468]: I see... but I'm not sure what I can do with it. Some lines use only spaces, some use TAB+2spaces and some only TABs.

Comment (choroba) [reply Makefile.PL:468]: 🤷🏽 select one style (probably the most common one) and apply it to the whole block? Maybe in a separate commit so it's clear nothing else changes.

Comment (pali) [reply Makefile.PL]: Ok, I tried something in new commit.

**Review by choroba — APPROVED** [review]

**Pull request merged**